sphinx/tutorial/cylc/runtime/introduction.rst

```diff
@@ -313,13 +262,65 @@ Where Do All The Files Go?
          :align: center
          :scale: 75%
 
+
+Running A Suite
```

Comment:
I agree that this section is slightly better placed a bit further down the 'Cylc Tutorial » Runtime » Introduction' main page, but it now comes under the 'Where Do All The Files Go?' high-level heading, & it doesn't really fit there. I think it might be clearer to put it just above 'Where Do All The Files Go?', perhaps grouping it with 'The Cylc GUI' as a section 'Running and Monitoring A Suite' or similar. I would also consider putting 'Suite Files' (excluding the practical if that would count as being part of this) at the top of the 'Where Do All The Files Go?' section so that the 'work' and 'job log' directories are introduced briefly on a general level with the other generated files before they are described in detail in sub-sections.

Reply:
The motive of this change is so that the start focuses on running tasks and the end on suites. Before we talked about tasks, then running suites, then back to tasks again. We can't order the content differently to the contents.

Reply:
Ah, I see. This motivation wasn't immediately obvious to me, but then again I didn't read the page from fresh. As long as there is a clear reason for placing certain sections in the given order, that is fine.